nasa/harmony
# HARMONY-1608: Added capability to provide default extend dimensions value for services support extend.

## Jira Issue ID
HARMONY-1608

## Description
Added capability to provide default extend dimensions value for services support extend.

## Local Test Steps
1. Under capabilities: in services.yml, for harmony-service-example in UAT, add:
```
extend: true
default_extend_dimensions: ['mirror_step']
```
2. Submit request: http://localhost:3000/C1233800302-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?granuleId=G1233800343-EEDTEST&format=image/tiff
3. Check the workflow_steps table, operation column to ensure the value `['mirror_step']` is passed via the extendDimensions property in the operation JSON
4. Submit request: http://localhost:3000/C1233800302-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?granuleId=G1233800343-EEDTEST&format=image/tiff&extend=a,b
5. Check the workflow_steps table, operation column to ensure the value `['a', 'b']` is passed via the extendDimensions property in the operation JSON

## PR Acceptance Checklist
